src/index.html

```diff
@@ -85,17 +95,20 @@ <h2 class="modal__title">Edit Profile</h2>
                 maxlength="200"
                 required
               />
-              <span id="profile-description-input-error" class="modal__error" >Please lengthen this text to 2 characters or more (you are currently usina 1 character).</span>
+              <span id="profile-description-input-error" class="modal__error"
+                >Please lengthen this text to 2 characters or more (you are
```

Comment:
Please, delete all initial error texts from `index.html` because they should appear only when the user starts typing anything in the inputs